docker/docker-compose.ci.yml

```diff
@@ -53,17 +56,20 @@ services:
     depends_on:
       - ganache-integration
       - unlock
+      - locksmith
     working_dir: /home/unlock/tests
   locksmith:
     environment:
       - DB_USERNAME
       - DB_PASSWORD
       - DB_NAME
       - DB_HOSTNAME
+      - NODE_ENV=production
```

Comment:
Hum. Is that required?

Reply:
We could probably come up with another environment for this, but the non-production environment will pull in the dev packages and will bring sqllite along with it.